### DISCUSSION TITLE
Is there any demo about multi-agent?

### DISCUSSION THREAD

**Discussion by JiayunjieJYJ** [allenai/ai2thor#947] [Q&A]
Is there any demo about multi-agent?

**Comment by laurimi** [comment]
Yes, here: https://colab.research.google.com/drive/1ityVEzHNN7k4Xek7-yfX_x_VRWNtO2wz?usp=sharing

**Reply by JiayunjieJYJ** [reply]
Thanks! I also have 3 questions:   
1. Does it support 2D lidar sensor ?  
2. Does it support URDF robot files?
3. How fast is the speed of get camera image?
